### DISCUSSION TITLE
gldf free and open for everyone?

### DISCUSSION THREAD

**Discussion by zergud** [globallightingdata/gldf#25] [General]
due to new Dialux Evo license policy: does "non-member" restrictions apply to luminaries prepared in gldf format ("Open and freely available for everyone." @ gldf.io)?

**Comment by bschmahl-dx** [comment]
This applies on all types of luminaires and file types (e.g. ULD, gldf, ldt, ies, cibse, ltli). 

Means that **gldf files of non-members will will not be supported**. 

If the manufacturer does not decide to become a member, users have the possibility to subscribe the DIALux pro feature "Unlock-Non Member". 

Details can be found here: https://www.dialux.com/en-GB/news-detail/dialux-evo-exclusively-with-luminaires-from-dialux-members

**Comment by zergud** [comment]
thank you!
IMHO, this contradicts the original idea of ​​creating the GLDF format
interesting to know Relux plans for using this format

**Comment by holg** [comment]
### Regarding Free and Open for Everyone GLDF, L3D, EULUMDAT & IES Tools

Regarding the discussion on whether GLDF is truly open: I have been working on a fully independent, open-source implementation that runs without any manufacturer restrictions.

This project is a proof-of-concept that the format is accessible. It is written in Rust and features a custom l3d 3D renderer and an egui-based interface.

The Stack (GLDF + L3D + LDT + IES): The viewer handles modern GLDF + L3D as well as legacy EULUMDAT and IES files. It now doesn't just parse data; it fully renders the 3D geometry.

Web App (WASM): https://gldf.icu/ (Runs entirely in browser)

iOS/Mac App: [GLDF Viewer on AppStore](https://apps.apple.com/de/app/gldf-viewer/id6755966812?l=en-GB)

Source & Desktop Binaries: https://github.com/holg/gldf-rs/releases (Binaries for Windows/Linux x64/arm64)

Note on Android: An Android version exists and is functional, but I haven't published it to the Play Store. As this is a non-profit hobby project, the administrative overhead for Google's closed testing/maintenance is currently too high.

Roadmap: Currently, this is a "weekend project," but the codebase is extensive. I am working on expanding the l3d renderer to support full scenes (Road, Room, Garden, Parking) rather than just single luminaires.
